packages/flutter/lib/src/cupertino/text_theme.dart

```diff
@@ -146,103 +133,93 @@ class CupertinoTextThemeData extends Diagnosticable {
-  final Brightness _brightness;
-  bool get _isLight => _brightness != Brightness.dark;
 
   final TextStyle _textStyle;
   /// Typography of general text content for Cupertino widgets.
-  TextStyle get textStyle => _textStyle ?? (_isLight ? _kDefaultLightTextStyle : _kDefaultDarkTextStyle);
+  TextStyle get textStyle => _textStyle ?? _defaults.textStyle;
 
   final TextStyle _actionTextStyle;
   /// Typography of interactive text content such as text in a button without background.
   TextStyle get actionTextStyle {
-    return _actionTextStyle ?? _kDefaultActionTextStyle.copyWith(
-      color: _primaryColor,
-    );
+    return _actionTextStyle ?? _defaults.actionTextStyle(primaryColor: _primaryColor);
   }
 
   final TextStyle _tabLabelTextStyle;
   /// Typography of unselected tabs.
-  TextStyle get tabLabelTextStyle => _tabLabelTextStyle ?? _kDefaultTabLabelTextStyle;
+  TextStyle get tabLabelTextStyle => _tabLabelTextStyle ?? _defaults.tabLabelTextStyle;
 
   final TextStyle _navTitleTextStyle;
   /// Typography of titles in standard navigation bars.
-  TextStyle get navTitleTextStyle {
-    return _navTitleTextStyle ??
-        (_isLight ? _kDefaultMiddleTitleLightTextStyle : _kDefaultMiddleTitleDarkTextStyle);
-  }
+  TextStyle get navTitleTextStyle => _navTitleTextStyle ?? _defaults.navTitleTextStyle;
 
   final TextStyle _navLargeTitleTextStyle;
   /// Typography of large titles in sliver navigation bars.
-  TextStyle get navLargeTitleTextStyle {
-    return _navLargeTitleTextStyle ??
-        (_isLight ? _kDefaultLargeTitleLightTextStyle : _kDefaultLargeTitleDarkTextStyle);
-  }
+  TextStyle get navLargeTitleTextStyle => _navLargeTitleTextStyle ?? _defaults.navLargeTitleTextStyle;
 
   final TextStyle _navActionTextStyle;
   /// Typography of interactive text content in navigation bars.
   TextStyle get navActionTextStyle {
-    return _navActionTextStyle ?? _kDefaultActionTextStyle.copyWith(
-      color: _primaryColor,
-    );
+    return _navActionTextStyle ?? _defaults.navActionTextStyle(primaryColor: _primaryColor);
   }
 
   final TextStyle _pickerTextStyle;
   /// Typography of pickers.
-  TextStyle get pickerTextStyle {
-    return _pickerTextStyle ??
-        (_isLight ? _kDefaultPickerLightTextStyle : _kDefaultPickerDarkTextStyle);
-  }
+  TextStyle get pickerTextStyle => _pickerTextStyle ?? _defaults.pickerTextStyle;
 
   final TextStyle _dateTimePickerTextStyle;
   /// Typography of date time pickers.
-  TextStyle get dateTimePickerTextStyle {
-    return _dateTimePickerTextStyle ??
-        (_isLight ? _kDefaultDateTimePickerLightTextStyle : _kDefaultDateTimePickerDarkTextStyle);
-  }
+  TextStyle get dateTimePickerTextStyle => _dateTimePickerTextStyle ?? _defaults.dateTimePickerTextStyle;
 
   /// Returns a copy of the current [CupertinoTextThemeData] with all the colors
   /// resolved against the given [BuildContext].
   CupertinoTextThemeData resolveFrom(BuildContext context, { bool nullOk = false }) {
```

Comment:
(I know this was already missing before you made your change, but:): It would be cool to document what the nullOk parameters does.